content/en/real_user_monitoring/guide/setup-feature-flag-data-collection.md

```diff
@@ -195,6 +195,112 @@ Amplitude does not support this integration. Create a ticket with Amplitude to r
 {{% /tab %}}
 {{< /tabs >}}
 
+
+### ConfigCat integration
+
+{{< tabs >}}
+{{% tab "Browser" %}}
+
+Initialize the ConfigCat Javascript SDK with subscribing to the `flagEvaluated` event that will report feature flag evaluations to Datadog:
```

Comment:
```suggestion
Initialize the ConfigCat Javascript SDK with subscribing to the `flagEvaluated` event that reports feature flag evaluations to Datadog:
```

Reply:
I also applied `by` instead of `with` to ensure consist texts.